contents/teams/growth/objectives.mdx

```diff
@@ -0,0 +1,34 @@
+## Q4 2025 objectives
+
+This team is now back online after a short time of having no assigned engineers to it - the old team did such a great job they now have their own product!
+
+This is also only a half quarter for this team, plus we're onboarding <TeamMember name="Matthew Brookes" photo />.
+
+
+### Cross-Sell
+
+<TeamMember name="Rafael Audibert" photo /> will be working on cross-sell motions to continue easing our customers into adopting more of our products - that's our moat.
+- Make use of the huge amount of data we capture for product intent signals inside PostHog.
+- Work with [Team Platform UX](/teams/platform-ux) to improve the sidebar experience for onboarding.
+- Expose PostHog signals to Array.
+- Stretch goal: Consolidate PostHog Ads concept (using the sidebar? using notifications? using email marketing?)
```

Comment:
What does this mean? What will you actually do? (I realize it's a side quest - but we should flesh it out, or make fleshing it out the goal, or remove it)